stac_fastapi/api/stac_fastapi/api/app.py

```diff
@@ -110,7 +110,7 @@ class StacApi:
             ),
             takes_self=True,
         ),
-        converter=update_openapi,
+        converter=update_openapi,  # type: ignore
```

Comment:
for some reason `attr + Mypy` doesn't like this (if we ever move to `attrs.field` it works fine